src/routes.js

```diff
@@ -171,18 +172,16 @@ export default (apollo, store) => {
       }
     },
     {
-      path: `/extension`,
-      name: `extension`,
-      components: {
-        session: () => import(`./components/common/TmSessionExtension`)
-      },
-      meta: {
-        feature: "Session"
+      path: `/extension/:address/:network`,
```

Comment:
it's odd to me that address comes before network

Reply:
hard to change now. but we can do it